RunSamples.cmd

```diff
@@ -37,17 +53,14 @@ set TEMP_DIR=%SPARKCLR_HOME%\Temp
 if NOT EXIST "%TEMP_DIR%" mkdir "%TEMP_DIR%"
 set SAMPLES_DIR=%SPARKCLR_HOME%\samples
 
-@echo JAVA_HOME=%JAVA_HOME%
-@echo SPARK_HOME=%SPARK_HOME%
-@echo SPARKCLR_HOME=%SPARKCLR_HOME%
-@echo SPARKCSV_JARS=%SPARKCSV_JARS%
+@echo [RunSamples.cmd] JAVA_HOME=%JAVA_HOME%
+@echo [RunSamples.cmd] SPARK_HOME=%SPARK_HOME%
+@echo [RunSamples.cmd] SPARKCLR_HOME=%SPARKCLR_HOME%
+@echo [RunSamples.cmd] SPARKCSV_JARS=%SPARKCSV_JARS%
 
 pushd %SPARKCLR_HOME%
-@cd
-
-@echo ON
 
+@echo [RunSamples.cmd] call %SPARKCLR_HOME%\scripts\sparkclr-submit.cmd --exe SparkCLRSamples.exe %SAMPLES_DIR% spark.local.dir %TEMP_DIR% sparkclr.sampledata.loc %SPARKCLR_HOME%\data %*
```

Comment:
It might be easy for maintenance if we extract `%SPARKCLR_HOME%\scripts...` to a variable here.